ingestion_tools/yarn.lock

```diff
@@ -0,0 +1,8 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
+allure@^0.0.0:
```

Comment:
I believe we wanted to not have this committed in for now.

Reply:
Sorry, removed!